package.json

```diff
@@ -7,14 +7,19 @@
     "hexo": "./bin/hexo"
   },
   "files": [
-    "lib",
+    "dist/**",
     "completion",
     "bin",
     "assets"
   ],
+  "types": "./dist/hexo.d.ts",
   "scripts": {
+    "prepublish ": "npm run clean && npm run build",
```

Comment:
Would you please add `npm install` here? I may forget to run it before publishing 😓 

```suggestion
    "prepublish ": "npm install && npm run clean && npm run build",
```